### DISCUSSION TITLE
Is musicblock-v4 supposed to look like this in development?

### DISCUSSION THREAD

**Discussion by Honey2339** [sugarlabs/musicblocks-v4#371] [General]
![image](https://github.com/sugarlabs/musicblocks-v4/assets/121949298/b2f2b851-c0df-414d-87ed-9105201656a6)

this might be a silly question but i tried to add the project file (project.html) from vanilla musicblock to the musicblock-v4 , and after importing the project nothing happened. So the project rn suppose to look like this?

**Comment by Karan-Palan** [comment]
Yes
